.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,32 @@
+name: Docker
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 5 * * FRI'
```

Comment:
Why the cron job?

Reply:
We are doing that currently in travis as well and `votca/buildenv` get updated a couple of hours earlier.

Reply:
Why not run it on push to master or so?

Reply:
Already enabled, but sometimes we don’t push for weeks.